Vegeting/Screens/GroupList/Component/ClubListCollectionViewCell.swift

```diff
@@ -18,18 +23,23 @@ final class ClubListCollectionViewCell: UICollectionViewCell {
         let label = UILabel()
         label.numberOfLines = 2
         label.tintColor = .black
+        label.font = .systemFont(ofSize: 16, weight: .regular)
```

Comment:
p2;

저희 폰트는 개발 레퍼런스 .preferredFont 함수를 통해
피그마 페이지에 나와있는 폰트로 적용해주시면 감사하겠습니다!

```suggestion
        label.font = .preferredFont(forTextStyle: .callout)
```

Reply:
[b2a2050](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/98/commits/b2a20506dec47a794f36628947f98722baf0b890)
적용완료!!